test/Microsoft.AspNet.Mvc.FunctionalTests/InputFormatterTests.cs

```diff
@@ -60,47 +60,6 @@ public InputFormatterTests(MvcTestFixture<FormatterWebSite.Startup> fixture)
         }
 
         [Theory]
-        [InlineData("", true)]
-        [InlineData(null, true)]
-        [InlineData("invalid", true)]
-        [InlineData("application/custom", true)]
-        [InlineData("image/jpg", true)]
-        [InlineData("", false)]
-        [InlineData(null, false)]
-        [InlineData("invalid", false)]
-        [InlineData("application/custom", false)]
-        [InlineData("image/jpg", false)]
-        public async Task ModelStateErrorValidation_NoInputFormatterFound_ForGivenContentType(
-            string requestContentType,
-            bool filterHandlesModelStateError)
-        {
-            // Arrange
-            var actionName = filterHandlesModelStateError ? "ActionFilterHandlesError" : "ActionHandlesError";
-            var expectedSource = filterHandlesModelStateError ? "filter" : "action";
-            var input = "{\"SampleInt\":10}";
-            var content = new StringContent(input);
-            content.Headers.Clear();
-            content.Headers.TryAddWithoutValidation("Content-Type", requestContentType);
-
-            // Act
-            var request = new HttpRequestMessage(HttpMethod.Post, "http://localhost/InputFormatter/" + actionName);
-            request.Headers.Accept.Add(MediaTypeWithQualityHeaderValue.Parse("application/json"));
-            request.Content = content;
-            var response = await Client.SendAsync(request);
-
-            var responseBody = await response.Content.ReadAsStringAsync();
-            var result = JsonConvert.DeserializeObject<FormatterWebSite.ErrorInfo>(responseBody);
-
-            // Assert
-            Assert.Equal(1, result.Errors.Count);
-            Assert.Equal("Unsupported content type '" + requestContentType + "'.",
-                         result.Errors[0]);
```

Comment:
Why remove this test? All that's changing here should be `ErrorMessage != null` -> `Exception != null`.